### PR TITLE
Fix app shutdown issue in sendbird.

### PR DESCRIPTION
**Pull request by lyw0149**
check network state and prevent sendbird when network is unconnected.

### CODE REVIEW

File: src/components/Chat/ChannelList.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ import {
   TextInput,
   Vibration,
   View,
+  NetInfo,
```
Comment (ace0625) [inline src/components/Chat/ChannelList.js]: Need to sort

File: src/components/Chat/ChannelList.js [hunk 2]
```diff
@@ -43,52 +44,73 @@ class ChannelList extends Component {
...
     }
   }
 
   renderRow(rowData) {
     return (
       <Row
-      _id = {this.state.me._id}
-      dataSource={rowData}
-    />
+        _id={this.state.me._id}
```
Comment (ace0625) [inline src/components/Chat/ChannelList.js]: should change without underscore.

File: src/components/Chat/ChannelList.js (hunk 1 above)
Comment (lyw0149) [reply src/components/Chat/ChannelList.js]: OK.

File: src/components/Chat/ChannelList.js (hunk 2 above)
Comment (lyw0149) [reply src/components/Chat/ChannelList.js]: Ok. Thanks

**Comment by ace0625** [comment]
looking good

**Comment by MechanicKim** [comment]
Okay. good job, Luke! LGTM

**Comment by ace0625** [comment]
Lgtm

**Pull request merged**